src/modules/commas/add.ts

```diff
@@ -17,7 +17,7 @@ const addCommas = (number?: number | string): string | undefined => {
 		? (digitsFaToEn(convertedToString) as string)
 		: convertedToString;
 
-	return tokenizedToEnglish.replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+	return tokenizedToEnglish.replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
![image](https://user-images.githubusercontent.com/9049092/111630559-bbb0b200-8807-11eb-9919-78a376e520d5.png)
I think we have to split the input by dot and then add commas to the first section and then merge them together. by this situation, we supports all other browsers.